website/docs/en/types/asset-info.mdx

```diff
@@ -0,0 +1,17 @@
+<>
```

Comment:
why do we need this fragment, it looks wired

Reply:
bug of rspress